taxcalc/functions.py

```diff
@@ -498,26 +503,30 @@ def StdDed(DSI, _earned, STD, e04470, e00100, e60000,
     c15200 = c04200
 
     # Compute the total standard deduction
+    _standard = c04100 + c04200
+
     if (MARS == 3 or MARS == 6) and (MIDR == 1):
         _standard = 0.
-    else:
-        _standard = c04100 + c04200
 
-    # ???
+    # c04500 = c00100 - max(c04470, max(c04100, _standard + e37717))
+    c04500 = c00100 - max(c21060 - c21040, _standard + e37717)
+
     if FDED == 1:
         _othded = e04470 - c04470
+        c04100 = 0.
```

Comment:
`FDED == 1` makes sense for `_standard = 0.` while `c04100, c04200 = 0., 0.` shall not be true. The latter part, however, has no impact at all.